### ISSUE TITLE
Dorn's 2015 paper.

### ISSUE THREAD

**Issue by clegoues** [squaresLab/genprog-code#6]
Graphics/shader thing; should be added?

**Comment by weimer** [comment]
> putative 

That was harsh, Tai. :-)

(1) 

http://web.eecs.umich.edu/~weimerw/p/dorn-pg2015-slides.pdf 

(2)

http://web.eecs.umich.edu/~weimerw/data/bandlimiting

**Comment by jlacomis** [comment]
I'm getting a 403 forbidden at http://web.eecs.umich.edu/~weimerw/data/bandlimiting

**Comment by weimer** [comment]
You are correct (currently). My request to umich to enable webserver-generated directory listings for my directories is already filed. 

If you need to see it now, replace web.eecs.umich.edu/~weimerw with www.cs.virginia.edu/~weimer .
